### PR TITLE
[SYNINF-188] Sane error handling for continuous work

### PR DESCRIPTION
**Pull request by kotchaosu**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -167,38 +168,75 @@ func updateNodesStats(config *Config) {
...
+
+	err = json.Unmarshal(file, c)
+	if err != nil {
+		log.Error(err)
+		panic(err)
+	}
+	return true
+}
+
+func runLoadConfigLoop(path string, c *Config) {
```
Comment (23doors) [inline main.go:220]: Why is it needed? How can loading a config file go wrong here?

**Comment by 23doors** [comment]
lgtm

File: main.go (hunk 1 above)
Comment (kotchaosu) [inline main.go:220]: This is step further. I want to make config hot-swap possible in near future.

Comment (23doors) [inline main.go:220]: Makes sense, that's what I thought but saw that config was only loaded once for now.

**Pull request merged**